src/omv/sensors/paj6100.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * This file is part of the OpenMV project.
+ * Copyright (c) 2019 Lake Fu <lake_fu@pixart.com>
+ * This work is licensed under the MIT license, see the file LICENSE for details.
+ *
+ * PAJ6100 driver.
+ *
+ */
+#ifndef __PAJ6100_H__
+#define __PAJ6100_H__
+#if (OMV_ENABLE_PAJ6100 == 1)
+#include <stdbool.h>
+#include "sensor.h"
+#define PAJ6100_XCLK_FREQ 6000000
+
+bool paj6100_detect(sensor_t *sensor);
+int paj6100_init(sensor_t *sensor);
+#endif
```

Comment:
```suggestion
#if (OMV_ENABLE_PAJ6100 == 1)
#define PAJ6100_XCLK_FREQ 6000000

bool paj6100_detect(sensor_t *sensor);
int paj6100_init(sensor_t *sensor);
```

Reply:
Ok, there is final time I try to .. discuss about this.
We should try to decoupling as much as possible in Header file. 
I can remove #include <stdbool..h> for paj6100.h but it make the user of paj6100.h to know that MUST include stdbool.h before include paj6100.h, WE SHOULD AVOID THAT.

I don't understand why we put us in danger, if one day this project grows up like Android or Linux lite-kernel, it is hard to debug a "header order issues".

Ok, that is my suggestion. 
It is your project, I will do what you want me to do.

But, think about it. It is good for this project.